internal/driver/helper.go

```diff
@@ -14,23 +14,17 @@ import (
 	"github.com/edgexfoundry/go-mod-core-contracts/v3/errors"
 )
 
-const (
-	redactedStr = "//<redacted>@"
```

Comment:
I feel this makes more sense being defined here, closer to its usage

Reply:
Ok, I thought it would be better to keep all constants in one place.